plotly/io/_renderers.py

```diff
@@ -394,6 +395,7 @@ def show(fig, renderer=None, validate=True, **kwargs):
 renderers['chrome'] = BrowserRenderer(config=config, using='chrome')
 renderers['chromium'] = BrowserRenderer(config=config, using='chromium')
 renderers['iframe'] = IFrameRenderer(config=config)
+renderers['sphinx'] = SphinxGalleryRenderer()
```

Comment:
Will this be useful in plain sphinx (without sphinx gallery)?  If not, lets name the renderer `'sphinx_gallery'`